Comparison Engine/Comparison Engine/Base Classes/Drink.cs

```diff
@@ -19,10 +19,11 @@ public Drink(int drinkID, string drinkName)
             this.drinkName = drinkName;
         }
 
-        //Changes the name of this drink object
-        public void EditDrinkName(string newName)
+        //Changes properties of this Drink Object
+        public void EditDrink(DrinkStruct drinkStruct)
         {
-            drinkName = newName;
+            if (drinkStruct.drinkName != null) this.drinkName = drinkStruct.drinkName;
+            if (drinkStruct.ingredientList != null) this.ingredientList = drinkStruct.ingredientList;
```

Comment:
For now, I believe there's not much integration of ingredientList and it cannot be edited or added, but probably will do something with it in the future, just a notice